Library/Homebrew/sorbet/rbi/upstream.rbi

```diff
@@ -2,21 +2,3 @@
 
 # This file contains temporary definitions for fixes that have
 # been submitted upstream to https://github.com/sorbet/sorbet.
-
-# https://github.com/sorbet/sorbet/pull/7650
```

Comment:
unrelated cleanup, while I'm in the `sorbet` dir (the linked upstream PRs have merged)